packages/mui-material/src/styles/excludeVariablesFromRoot.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * @internal These variables should not appear in the :root stylesheet when the `defaultMode="dark"`
+ */
+const excludeVariablesFromRoot = (cssVar: string) =>
```

Comment:
I am wondering if people will ever need this as feature on their side. If yes, we could export this so that they can extend/override it.